docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+# https://docs.docker.com/compose/
+
+version: "3.6"
+
+
+# To build all run "docker-compose up"
+services:
+
+  # This builds vanilla based on dockerfile in this dir
+#  vanilla:
+#    build: .
```

Comment:
Do you want this piece for vanilla docker commented out?  Might want to update the dev-mode.md instructions for how to run all this as a developer.  I generally just need to run the postgress docker itself right?